### ISSUE TITLE
Difference to original mod?

### ISSUE THREAD

**Issue by DE-Schmitt** [jetelain/cTab#33]
Hey @jetelain ! 

What is the difference between your project and the original https://github.com/Riouken/cTab/tree/master/%40cTab/addons/cTab ?

Your projects splits the ctab into ctab_core.pbo, ctab_ctab.pbo, ctab_main.pbo while the original only has ctab.pbo. Why do you split it into three pbos?

I want to integrate https://ctab.plan-ops.fr/ into my github. Thus it seem I need yours instead of the original ctab. But I am curious whats the differences.

Greetings!

**Comment by DE-Schmitt** [comment]
It is quite obvious that dragging of markers is missing in this mod. Is there a technical limitation that hinders you from implementing ALT+Mouse Drag for moving already placed markers? Currently you have to delete the marker and then create one from scratch, whenever the marked enemy units have moved somewhere else.

**Comment by jetelain** [comment]
Hi,
The changes are listed here : https://github.com/jetelain/cTab/blob/master/CHANGELOG.md (look at 2.5, 2.6 and 2.7).

I have reorganized pbo to make developpement and debugging easier (with hemtt and file patching). I plan to have a pbo per device (tablet, android, etc.)

I did not notice that orginal mod allowed to move markers, it might have been broken when I've moved to CBA key binding. I will have a look to fix this / add the feature again.
